pkg/apis/sources/v1/apiserver_types.go

```diff
@@ -100,6 +100,10 @@ type ApiServerSourceStatus struct {
 
 	// Namespaces show the namespaces currently watched by the ApiServerSource
 	Namespaces []string `json:"namespaces"`
+
+	// Auth provides the relevant information for OIDC authentication.
+	// +optional
+	Auth *duckv1.AuthStatus `json:"auth,omitempty"`
```

Comment:
This is not needed here. We already have the `AuthStatus` in the `duckv1.SourceStatus`